Userland/Applications/Calendar/main.cpp

```diff
@@ -16,24 +17,31 @@
 #include <LibGUI/Icon.h>
 #include <LibGUI/Menu.h>
 #include <LibGUI/Menubar.h>
+#include <LibGUI/Painter.h>
 #include <LibGUI/Process.h>
 #include <LibGUI/Toolbar.h>
 #include <LibGUI/Window.h>
+#include <LibGfx/Font/FontDatabase.h>
+#include <LibGfx/Palette.h>
 #include <LibMain/Main.h>
 
-ErrorOr<int> serenity_main(Main::Arguments arguments)
+using namespace Calendar;
```

Comment:
Can't we do a 
```cpp
namespace Calendar {
...
}
```

Reply:
I don't want the `serenity_main` in the namespace, but I want to use the `Calendar` namespace so I don't have to do `Calendar::CalendarWidget`

Reply:
I feel like Calendar::CalendarWidget is fine. Using the namespace at the top of the file like that can make it unclear at first glance where things are coming from